pkg/node/id.go

```diff
@@ -46,20 +43,20 @@ func IDFromString(s string) *ID {
 
```

Comment:
can we remove this commented out code?

Reply:
Ah, this must have been uncommented in a merge, I just removed it and pushed up. I'll make sure this build passes and then merge! 👍